mods/sbz_meteorites/init.lua

```diff
@@ -50,7 +50,27 @@ local function meteorite_globalstep(dtime)
     storage:set_float("time_since_last_spawn", time_since)
 end
 
-minetest.register_globalstep(meteorite_globalstep)
+local G = 1/1000
+local max_attraction = 3
+minetest.register_globalstep(function(dt)
+	meteorite_globalstep(dt)
+	for _, player in ipairs(minetest.get_connected_players()) do
+		local nodes, _ = minetest.find_nodes_in_area(vector.subtract(player:get_pos(), vector.new(10,10,10)), vector.add(player:get_pos(), vector.new(10,10,10)), {"group:gravity"})
```

Comment:
Every globalstep doing this will be expensive doing this, you then do it again. I haven't profiled this but my experience is this will be brutal.

Reply:
yes i know it's inefficient i was too lazy to make a system that stores all gravity nodes and adds force based on that but you're welcome to do that